backend/app/src/chat/chat.gateway.ts

```diff
@@ -0,0 +1,52 @@
+import {
+  ConnectedSocket,
+  MessageBody,
+  SubscribeMessage,
+  WebSocketGateway,
+  WebSocketServer,
+} from '@nestjs/websockets'
+import { Server, Socket } from 'socket.io'
+import { chatEvent } from 'src/configs/chatEvent.constants'
+import { ChatMessageDto } from './chat.dto'
+
+@WebSocketGateway({ namespace: 'chat', cors: true })
+export class ChatGateway {
+  @WebSocketServer()
+  server: Server
+
+  clients = []
+
+  afterInit(server: Server) {
+    console.log('chat:', 'new server')
+  }
+
+  handleConnection(client: Socket) {
+    console.log('chat:', 'new connection')
+  }
+
+  handleDisconnect(client: Socket) {
+    console.log('chat:', 'disconnected')
+  }
+
+  @SubscribeMessage(chatEvent.JOIN)
+  onJoinRoom(
+    @ConnectedSocket() client: Socket,
+    @MessageBody() data: ChatMessageDto,
+  ) {
+    const { senderId, roomId } = data
+    client.join(roomId)
+    console.log('chat:', senderId + ' has entered to ' + roomId)
+    this.server.to(roomId).emit(chatEvent.NOTICE, senderId + ' has entered!!')
```

Comment:
```suggestion
    console.log(`chat: ${senderId} has entered to ${roomId})
    this.server.to(roomId).emit(chatEvent.NOTICE, `${senderId} has entered!!`)
```
[템플릿 리터럴](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Template_literals)로 바꾸면 좋을 것 같습니다

Reply:
확인했습니다. 다음 커밋때 반영하겠습니다.